Arbie & Yin - Solved Issue #4

arbie333 opened the pull request:
We edited tests for the cflat compiler so that it passed all the tests when we run `make test`.
```
Basic compiler setup
  ✔ Should not crash on instantiation

cflatp parsing
  ✔ Parses simple class with one method
```

arbie333 committed:
Update cflat-tests.ts

geniuseva committed:
Update default.lir

Fix a problem of forcing a new line at the end of the code in any browser.

geniuseva committed:
test

test

geniuseva committed:
test 2

test 2

arbie333 committed:
Passed cflat test

arbie333 committed:
Merge branch 'cflat-language' of https://github.com/maemre/compiler-explorer into cflat-language

maemre commented:
This all makes sense and looks good to me, except the name of this test and the suite it's in:

```
cflatp parsing
  ✔ Parses simple class with one method
```

Please use meaningful names for this test (the test is not for parsing, and there are no classes or methods in Cflat, you can say "compiles a simple LIR program" instead), then you can merge this. I'm approving it ahead of time to speed up merging it.

maemre reviewed (approved):
See my notes on the PR about the test naming convention. Otherwise, this looks.

geniuseva committed:
Update cflat-tests.ts

1. changed "cflatp parsing" to "cflat compiling"
2. changed "Parses simple class with one method" to "Compiles a simple LIR program"

geniuseva commented:
1. changed "cflatp parsing" to "cflat compiling"
2. changed "Parses simple class with one method" to "Compiles a simple LIR program"
